codegen-on-oss/codegen_on_oss/context_server/server.py

```
@@ -0,0 +1,203 @@
+
+@app.get("/")
+async def root():
+    """Root endpoint."""
+    return {"message": "Welcome to the CodeContextRetrievalServer"}
+
+
+@app.post("/analyze")
+async def analyze_repository(request: RepositoryRequest):
+    """
+    Analyze a repository and return the results.
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)